### DISCUSSION TITLE
Welcome to ParaBlade Discussions!

### DISCUSSION THREAD

**Discussion by NAnand-TUD** [NAnand-TUD/parablade#8] [Announcements]
# 👋 Welcome to ParaBlade!

More and more people are using ParaBlade, and this is very motivating for us developers. We open this discussion page for users/developers to ask questions or request features. Let's engage...

**Comment by yousufiitm** [comment]
Hi..
I am unable to install in python pip
says 
The system cannot find the file specified while executing command git clone -q https://github.com/NAnand-TUD/parablade.git

can u help with this
